app/src/sharedTest/java/org/oppia/app/faq/FAQFragmentTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.app.faq
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers.not
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher
+
+@RunWith(AndroidJUnit4::class)
+class FAQFragmentTest {
+
+  var itemCount: Int = 9
+
+  @Test
+  fun openFAQActivity_scrollRecyclerViewToZeroPosition_checkFeaturedQuestionsDisplayedSuccessfully() {
+    launch(FAQActivity::class.java).use {
+      onView(withId(R.id.faq_fragment_recycler_view)).perform(
+        RecyclerViewActions.scrollToPosition<RecyclerView.ViewHolder>(
```

Comment:
Update this to `scrollToPosition<RecyclerView.ViewHolder>(` here and everwhere else and remove unused imports.

Also, please correct the indentation for entire file. You can do this by first bringing the relative code to a single line rather than multiple lines and then press `Control + Alt + L`

Reply:
Done :)